src/vsg/utils/GraphicsPipelineConfigurator.cpp

```diff
@@ -151,7 +151,7 @@ bool DescriptorConfigurator::assignTexture(const std::string& name, const ImageI
 
 bool DescriptorConfigurator::enableBuffer(const std::string& name)
```

Comment:
Shouldn't this be enableDescriptor(..)?

Reply:
no. same reason as for in `GraphicsPipelineConfigurator` since we want to differentiate between buffers (uniform or storage) and images (textures).